src/poll.ts

```diff
@@ -236,17 +250,53 @@ namespace Poll
 
 			return null;
 		}
+		
+		if (!voicePoll) 
+		{
+			voiceChannel = null;
+		}
 
 		return new Poll
 		(
 			message,
 			desc + ' ' + command[1],
 
 			() => action(target),
-			() => (target.voiceChannelID === message.member.voiceChannelID),
-			() => Math.floor(voiceChannel.members.array().length * fraction)
+			() => (true),
+			() => 
+			{
+				let userCount: number = 0;
+				if (voicePoll)
+				{
+					userCount = voiceChannel.members.array().length;
```

Comment:
Bots can join voice channels too. This is actually a problem with the bot I didn't realize until now.